molecule/default/verify.yml

```diff
@@ -189,3 +189,21 @@
       msg: |
         'ms' not found in output of 'dokku apps:list':
         {{ dokku_apps.stdout }}
+  # Testing dokku_builder
```

Comment:
If you have the time, adding a `global` test could be useful (I guess that should have errored with the current implementation)